client/src/components/job-application-component/applicant-information-card/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  Card,
+  CardContent,
+  Typography,
+} from "@mui/material";
+import { Divider } from "../index.styles";
+
+import BoxRowComponent from "../box-row-component";
+
+function ApplicantInformationCard({ application }) {
+
+  const [applicationName] = useState(application.name);
+  const [applicationEmail] = useState(application.email);
+  const [applicationPhoneNumber] = useState(application.phone);
+  const [applicationPostalCode] = useState(application.postal_code);
+  const [applicationStatus] = useState(application.status_in_canada);
+
+  const emailWrapper = (email) => {
+    return <a href={`mailto:${email}`}>{email}</a>;
+  };
+
+  return (
+    <Card
+      style={{
+        width: "50%",
+        borderRadius: 8,
+        boxShadow: 3,
+        border: "1px solid #e0e0e0",
+      }}
+      sx={{
+        mr: 2,
+        ml: 9,
+      }}
+    >
+      <CardContent
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+        }}
+      >
+        <Typography
+          variant="h5"
+          align="left"
+          gutterBottom
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            margin: 0,
+          }}
+        >
+          Applicant Information
+        </Typography>
+      </CardContent>
+      <Divider />
+      <CardContent>
+        <BoxRowComponent
+            leftSide="Name"
+            rightSide={applicationName}
+          />
+          <BoxRowComponent
+            leftSide="Email"
+            rightSideWrapper={emailWrapper}
```

Comment:
This wrapper doesn't seem to be a prop used in the BoxRowComponent, and as a result your email isn't getting included as a mailto link